lumimqtt/__main__.py

```diff
@@ -51,6 +51,7 @@ def main():
         'sensor_threshold': 50,  # 5% of illuminance sensor
         'sensor_debounce_period': 60,  # 1 minute
         'light_transition_period': 1.0,  # second
+        'light_tele_period': 60, # 1 minute
```

Comment:
What does `tele` mean in this case?

Reply:
I'm not a native English speaker. But I've seen in multiple projects and contexts the use of 'tele' short word for [telemetry](https://www.britannica.com/technology/telemetry). I've seen it in tasmota and other contexts. That's why I named it like that.

But I could change it to something like frequency if you like it better.

Reply:
This is not telemetry as I can see. What about changing it to a more clear `light_notification_period`?

Reply:
Sure thing will change it right away.

Reply:
Changes have been made 😄